src/pages/index.js

```diff
@@ -1,179 +1,265 @@
 
-const cardListEl = document.querySelector(".cards__card-list");
+// CARD VARIABLES
 
-// Buttons etc.
-const addNewCardButton = document.querySelector(".profile__add-button");
-const profileModalCloseButton = profileEditModal.querySelector(
-  "#profile-close-button"
-);
-const addCardModalCloseButton = addCardModal.querySelector(
-  "#modal-close-button"
-);
-const profileEditButton = document.querySelector("#profile-edit-button");
+const addCardModal = document.querySelector("#add-card-modal");
+const addCardForm = addCardModal.querySelector("#add-card-form");
+const cardsWrap = document.querySelector(".cards__list");
 
-// Inputs
-const profileNameInput = document.querySelector("#profile-name-input");
-const profileDescriptionInput = document.querySelector(
-  "#profile-description-input"
-);
-const cardTitleInput = addCardFormElement.querySelector(
-  ".modal__input_type_title"
-);
-const cardUrlInput = addCardFormElement.querySelector(".modal__input_type_url");
-const previewImageModal = document.querySelector("#preview-image");
-const previewImageClose = document.querySelector("#preview-image-close");
+// PREVIEW VARIABLES
 
-const cardSelector = "#card-template";
+const previewModal = document.querySelector("#preview-modal");
+const modalImage = previewModal.querySelector(".modal__image");
 
-/*------------------------------------------------*/
-/*                  Validation                    */
-/*------------------------------------------------*/
+// BUTTONS
 
-const config = {
-  formSelector: ".modal__form",
-  inputSelector: ".modal__input",
-  submitButtonSelector: ".modal__button",
-  inactiveButtonClass: "modal__button_disabled",
-  inputErrorClass: "modal__input_type_error",
-  errorClass: "modal__error_visible",
-};
+const profileEditBtn = document.querySelector("#profile-edit-button");
+const addNewCardBtn = document.querySelector("#add-card-button");
 
-const editFormValidator = new FormValidator(config, profileEditForm);
-const addFormValidator = new FormValidator(config, addCardFormElement);
+// API'S
 
-editFormValidator.enableValidation();
-addFormValidator.enableValidation();
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "8591f7fe-0114-4a2f-84de-cfb13be7817c",
+    "Content-Type": "application/json",
+  },
+});
 
-const formValidators = {};
+api
+  .getInitialCards()
+  .then((cards) => {
+    newCardSection.setItems(cards);
+    newCardSection.renderItems();
+  })
+  .catch((err) => {
+    alert(`${err} Failed to get cards.`);
+  });
 
-// enable validation
-// const enableValidation = (config) => {
-//   const formList = Array.from(document.querySelectorAll(config.formSelector));
-//   formList.forEach((formElement) => {
-//     const validator = new FormValidator(config, formElement);
-//     // here you get the name of the form
-//     const formName = formElement.getAttribute("name");
+api
+  .getUserInfo()

```

Comment:
Better to use `Promise.all` to load initial data

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all